### PR TITLE
ejs/location location 2

### PR DESCRIPTION
**Pull request by jmtroth0**
Hey @millerdev, Jenny mentioned to me that you could be a good person to ping on some of these externalizing javascript prs while she is offline this week. They are [somewhat formulaic](https://github.com/dimagi/commcare-hq/blob/master/scripts/externalize_js.sh), but there is definitely some context around how some modules are used that I am ignorant of, which could cause some problems with some migrations.

Let me know if you would prefer not though. I'm sure they could also wait until Jenny is back next week.

### CODE REVIEW

File: corehq/apps/locations/static/locations/spec/location_drilldown_spec.js [hunk 1]
```diff
@@ -1,9 +1,6 @@
 /*eslint-env mocha */
-/*global _ LocationSelectViewModel */
 
-
-/*eslint no-unused-vars:0*/
-var LOAD_LOCS_URL = '/locations/api/';
+var LocationSelectViewModel = hqImport('locations/js/location_drilldown').LocationSelectViewModel;
```
Comment (millerdev) [inline corehq/apps/locations/static/locations/spec/location_drilldown_spec.js]: would it work to move this inside the `describe` block below so as not to assign it to a global variable?

**Review by millerdev — COMMENTED** [review]
Only one comment. Otherwise looks good.

What's your feeling for the level of testing that you've done on this? Do you feel confident that the changes you've made will not break anything if we deploy it?

File: corehq/apps/locations/static/locations/spec/location_drilldown_spec.js (hunk 1 above)
Comment (jmtroth0) [reply corehq/apps/locations/static/locations/spec/location_drilldown_spec.js]: Yup, that should work. I had just been mirroring where it was before

**Comment by jmtroth0** [comment]
I would say that I am moderately confident of that. I went to the pages and the only errors that pop up also pop up on master. To be fair though, I don't have complex data locally, so it might not work

**Review by millerdev — APPROVED** [review]

**Pull request merged**